aws/retry/standard.go

```diff
@@ -51,8 +51,11 @@ var DefaultRetryableHTTPStatusCodes = map[int]struct{}{
 // DefaultRetryableErrorCodes provides the set of API error codes that should
 // be retried.
 var DefaultRetryableErrorCodes = map[string]struct{}{
+	"RequestExpired":          {},
```

Comment:
Thinking we should probably _just_ use these to do the `probSkew` wrapping instead of adding them to the default policy.

Reply:
I see your point. Was debating between keeping all skew stuff together. Moved them and expanded the probSkew logic to include this